### ISSUE TITLE
New post with default "settings"

### ISSUE THREAD

**Issue by gboone** [18F/federalist#174]
On 18f.gsa.gov we require blog posts to have a title, authors, tags, and a few other things. It'd be handy if there were a button somewhere in the edit interface that adds a new document in a folder with any required settings added at post creation.

The UX for a site owner might be having an interface for adding required settings on a per-collection basis (or something).

The UX for the editor might be:
1. Click to add a new post
2. Fill in a webform with any required settings
3. Go right into editing the content

I'd be interested in helping out with this, I've done some work with validating frontmatter in Jekyll. This seems somewhat dependent on #166 (or at least lower priority).

**Comment by dhcole** [comment]
@jeremiak good to close or should more be done before this is settled?

**Comment by dhcole** [comment]
Not sure if #228 would be considered a blocker here.
